packages/backend/migrations/20230731141948_create_fee_table.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('fees', function (table) {
+    table.uuid('id').primary()
+    table.uuid('assetId').references('assets.id').notNullable()
+    table.enum('type', ['SENDING', 'RECEIVING']).notNullable()
+    table.timestamp('createdAt').defaultTo(knex.fn.now())
+    table.timestamp('activatedAt')
```

Comment:
Initially I just had `createdAt` but changed it after this discussion in the issue https://github.com/interledger/rafiki/issues/1637

Reply:
I missed that. Thank you! However, We don't bother with any other business logic and I wonder if we should then also be that paranoid in the auth server because the grants also don't have an `activatedAt` field. @mkurapov

Reply:
see https://github.com/interledger/rafiki/pull/1702#discussion_r1294905765